src/spinline.jquery.js

```diff
@@ -78,7 +78,8 @@
                                 position: 'absolute',
                                 left: 0,
                                 width: 0,
-                                boxShadow: "2px 0 20px " + settings.color
+                                boxShadow: "2px 0 20px " + settings.color,
+				'z-index': 2147483647
```

Comment:
I tried to apply your spinline to element "header". But in my case line displayed incorrect because "z-index" has in the style one of elements header.